build-aux/snap/snapcraft.yaml

```diff
@@ -287,7 +287,7 @@ parts:
              ;;
            *)
              export CGO_ENABLED=1
-             GO_LD_FLAGS=()
+             GO_LD_FLAGS=(-buildmode=pie)
```

Comment:
I have a feeling this may blow up somewhere. Could we enable it for only one arch today and then explore adding it to subsequent arches as a followup?

Reply:
They are all pie already. It was only risc-v that was not.

Reply:
Ah maybe not on 32 bits architectures.

Reply:
It was only armhf, I have disabled it for that arch.